r/tests/testthat/test-json.R

```diff
@@ -40,13 +41,13 @@ test_that("Can read json file with scalars columns (ARROW-5503)", {
   tib <- as.data.frame(tab1)
   expect_equal(tib$hello, c(3.5, 3.25, 3.125, 0))
   expect_equal(tib$world, c(FALSE, NA, NA, TRUE))
+  skip_on_os("windows") # TODO: debug UTF-8 test
```

Comment:
JIRA issue?

Reply:
Good idea but I think @fsaintjacques has resolved the underlying issue on some docker-r build fix he was working on today, so (once this is in master) he can remove the skip on his branch.